### PR TITLE
now with completed assignment

### PR DESCRIPTION
**Pull request by bennettatoms**
{
completeness: 5,
comfortability: 4,
questions: n/a
}

### CODE REVIEW

File: seeds.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# seeds.sql that inserts some of your favorite TV shows into the shows table.
```
Comment (drmikeh) [inline seeds.sql]: Should use -- instead of # for SQL comments.

File: create_schema.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE new_shows (
```
Comment (drmikeh) [inline create_schema.sql]: table name should be "shows"

**Comment by drmikeh** [comment]
I added 2 minor comments, but it looks good.
:+1:

**Comment by bennettatoms** [comment]
Sorry, I created new_shows after the fact to troubleshoot the integer range problem and forgot to change it back. Corrected now, and sql comment format noted. Thanks!

**Pull request merged**